composition/src/client_component.cpp

```diff
@@ -14,6 +14,7 @@
 
 #include "composition/client_component.hpp"
 
+#include <inttypes.h>
```

Comment:
Technically should be `<cinttypes>` because this is c++ but it doesn't really matter.